components/upload/events.md

```diff
@@ -424,12 +414,9 @@ The `OnProgress` event fires each time a particular file makes a progress in its
 
 @[template](/_contentTemplates/common/general-info.md#event-callback-can-be-async)
 
-
-
-
 ## OnSuccess
 
-The `OnSuccess` event fires each time a particular request is successful - either an upload of a file, or the deletion of a file. The event arguments provide the file that was uploaded or deleted, and the type of the operation. The `Request` object in the event arguments also carries information about the server response - such as a status code and any custom messages the server returned.
+The `OnSuccess` event fires each time a particular request is successful - either an upload of a file, or the deletion of a file. The event arguments provide the file that was uploaded or deleted, and the type of the operation. The `Request` object in the event arguments also carries information about the server response - such as a status code and any custom messages the server returned. For example, this response can be used to obtain a URL string from the endpoint and show a preview of an uploaded image.
```

Comment:
I'd personally mention this when we have a real solution for it - either through a FileSelect component, or a sample project to link.

Reply:
This makes sense, but this information was already present, and also, people are asking for it, so I guess it better stays.